src/main/java/views/blackjack/blackjack.ftl.html

```diff
@@ -123,6 +123,47 @@
      });
 
 });
+
+//Function to handle hit requests from the user
+//Checks if the field of the hit button is "Hit"
+//If true then we make a post request to get a new card
+//Grab this new card, string configure it, and append
+//the new card to the users hand
+$("#dealHit").click(function(){
+    val = "";
+    data = "";
+    data = $("#dealHit").text();
+
+    if(data == "Hit"){
+        var $newCard;
+        //Send game data to get card{amount}
+        $.ajax({
+            type: "POST",
+            url: "/hit",
+            data: JSON.stringify(game),
+            success: function(data, status){
+                console.log("Data: " + data + "\nStatus: " + status);
+                //Display game data
+                display(data);
+                game = data;
+                //Loops through and rebuilds the entire deck
```

Comment:
I think what you meant is rebuilds the users hand. In the following code it actually just keeps adding the same cards to the users hand because you never run `$("#userCards").html("");` but that's an easy fix.